src/components/FixedGrid/FixedGridRow.tsx

```diff
@@ -24,27 +26,24 @@ export { GutterSize };
  * A `FixedGridRow` represents each row inside a `FixedGrid`. It should wrap `FixedGridColumn`s.
  */
 export default class FixedGridRow extends React.Component<FixedGridRowProps> {
-  public static defaultProps = {
-    gutterSize: GutterSize.SMALL,
-  };
-
   public render() {
-    const { children } = this.props;
-
-    return <div className={this.getClasses()}>{children}</div>;
+    const { className, bottomSpacing, gutterSize = GutterSize.SMALL } = this.props;
+    const classNames = getClassNames({ bottomSpacing, gutterSize });
+    return (
+      <div className={join(['y-fixedGridRow', className, classNames.root])}>
+        {this.childrenWithClassName(classNames.column)}
+      </div>
+    );
   }
 
-  private getClasses() {
-    const { bottomSpacing, className, gutterSize } = this.props;
-
-    const classes: string[] = ['y-fixedGridRow', `y-fixedGridRow__gutter-${gutterSize}`];
-    if (bottomSpacing) {
-      classes.push(`y-fixedGridRow__bottomSpacing-${bottomSpacing}`);
-    }
-    if (className) {
-      classes.push(className);
-    }
-
-    return classes.join(' ');
-  }
+  private childrenWithClassName = (className: string) => {
```

Comment:
this is pretty ugly. can we just move the margin styles to a selector on the parent row?

Reply:
Good idea.  I'll try.